### ISSUE TITLE
Xiaomi Mi Robot Vacuum Mop G1 charging/low battery issue mijia.vacuum.v2

### ISSUE THREAD

**Issue by robjampar** [merdok/homebridge-miot#272]
https://github.com/merdok/homebridge-miot/commit/ed7ca3cae6f2080de895abcd0196852bc7f79caa

I have a mijia.vacuum.v2 that always reports as "Battery Low" (both dock and vacuum)

here are the homebridge logs

[18/05/2022, 14:48:52] [homebridge-miot] [Bedroom 5 Vacuum] Model known: mijia.vacuum.v2!
[18/05/2022, 14:48:52] [homebridge-miot] [Bedroom 5 Vacuum] Initializing device!
[18/05/2022, 14:48:52] [homebridge-miot] [Bedroom 5 Vacuum] Initializing device services
[18/05/2022, 14:48:52] [homebridge-miot] [Bedroom 5 Vacuum] Device services: [
  "vacuum",
  "battery",
  "alarm",
  "identify",
  "filter",
  "brush-cleaner",
  "brush-cleaner",
  "map",
  "remote-control",
  "clean-record",
  "language",
  "go-charging",
  "other-status"
]
[18/05/2022, 14:48:52] [homebridge-miot] [Bedroom 5 Vacuum] Initializing device properties
[18/05/2022, 14:48:52] [homebridge-miot] [Bedroom 5 Vacuum] Device properties: [
  "vacuum:status",
  "vacuum:fault",
  "vacuum:mode",
  "vacuum:target-water-level",
  "vacuum:fan-level",
  "battery:battery-level",
  "battery:charging-state",
  "alarm:alarm",
  "alarm:volume",
  "filter:filter-life-level",
  "filter:filter-left-time",
  "brush-cleaner:brush-life-level",
  "brush-cleaner:brush-left-time",
  "brush-cleaner2:brush-life-level",
  "brush-cleaner2:brush-left-time",
  "map:points",
  "map:map-switch",
  "remote-control:direction-key",
  "clean-record:clean-area",
  "clean-record:clean-time",
  "clean-record:total-clean-area",
  "clean-record:total-clean-time",
  "clean-record:total-clean-count",
  "clean-record:current-clean-record",
  "language:language",
  "language:not-disturb-switch",
  "language:not-disturb-time",
  "language:time-zone",
  "other-status:mop-status",
  "other-status:log-str"
]
[18/05/2022, 14:48:52] [homebridge-miot] [Bedroom 5 Vacuum] Initializing device actions
[18/05/2022, 14:48:52] [homebridge-miot] [Bedroom 5 Vacuum] Device actions: [
  "vacuum:start-sweep",
  "vacuum:stop-sweeping",
  "vacuum:start-charge",
  "identify:identify",
  "filter:reset-filter-life",
  "brush-cleaner:reset-brush-life",
  "brush-cleaner2:reset-brush-life",
  "go-charging:start-charging",
  "go-charging:stop-charging"
]
[18/05/2022, 14:48:52] [homebridge-miot] [Bedroom 5 Vacuum] Successfully created a RobotCleaner device! It is a Xiaomi Mi Robot Vacuum Mop G1.
[18/05/2022, 14:48:52] [homebridge-miot] [Bedroom 5 Vacuum] Event with name clean-record:current-clean-record already exists. Not creating!
[
[18/05/2022, 14:48:52] [homebridge-miot] [Bedroom 5 Vacuum] Initializing accessory!
[18/05/2022, 14:48:52] [homebridge-miot] [Bedroom 5 Vacuum] Accessory successfully initialized!
[18/05/2022, 14:48:52] [homebridge-miot] [Bedroom 5 Vacuum] Registering 1 accessories!
[18/05/2022, 14:48:52] [homebridge-miot] [Bedroom 5 Vacuum] Everything looks good! Initiating property polling!

**Comment by robjampar** [comment]
![0B17093A-378D-4C80-92CB-75C453412A44](https://user-images.githubusercontent.com/7254610/169062003-0efc7a11-0026-403e-9fd8-1b1ae6284302.png)
![769B59FD-ABD6-4D79-BE14-7B3F430D2004](https://user-images.githubusercontent.com/7254610/169062022-76c22155-51f7-4efb-9bb6-cf1ad4515230.png)

**Comment by merdok** [comment]
Yeah, some devices work better over the cloud so sometimes it makes sense to force a device over the cloud even if it should actually work with a local connection.
